src/TLuaInterpreter.cpp

```diff
@@ -6780,6 +6799,16 @@ int TLuaInterpreter::exists( lua_State * L )
             it1++;
         }
     }
+    //TODO: exists(keybind) doesn't currently work, figure out why
+    else if( type == "keybind")
+    {
+        QMap<QString, TKey *>::const_iterator it1 = pHost->getKeyUnit()->mLookupTable.find( name );
+        while( it1 != pHost->getKeyUnit()->mLookupTable.end() && it1.key() == name )
```

Comment:
:thinking: If, as I suspect, `pHost->getKeyUnit()->mLookupTable` __is__ a `QMap<QString, TKey *>` can not the iteration loop be collapsed to:

   cnt += pHost->getKeyUnit()->mLookupTable.count( name );

There is an `(int)QMap::count(const Key &key) const` method after all...